frontend/lib/geo-autocomplete.tsx

```diff
@@ -244,8 +244,8 @@ export class GeoAutocomplete extends React.Component<GeoAutocompleteProps, GeoAu
   render() {
     return (
       <GeoDownshift
-        onChange={this.props.onChange}
-        defaultSelectedItem={this.props.initialValue}
+        onChange={item => item && this.props.onChange(item)}
+        initialSelectedItem={this.props.initialValue}
```

Comment:
According to the [v3.0.0 release notes](https://github.com/downshift-js/downshift/releases/tag/v3.0.0), `defaultSelectedItem` is now `initialSelectedItem`.